distribution/src/config/opensearch.yml

```
@@ -121,3 +121,9 @@ ${path.logs}
 # index searcher threadpool.
 #
 #opensearch.experimental.feature.concurrent_segment_search.enabled: false
+#
+#
+# Gates the optimization of datetime formatters caching along with change in default datetime formatter
+# Once there is no observed impact on performance, this feature flag can be removed.
+#
+#opensearch.experimental.optimization.datetime_formatter_caching.enabled: true
```

Since this is `true` by default, we could document that

```suggestion
#opensearch.experimental.optimization.datetime_formatter_caching.enabled: false
```

---

Done, thanks for pointing out